### PR TITLE
Add Operations in inventory explorer presentation

### PR DESCRIPTION
**Pull request by openwarrior**
- Add Operation model
- Present operations in explorer associated with resources

### CODE REVIEW

File: mobile/src/main/java/org/hawkular/client/android/backend/BackendClient.java [hunk 1]
```diff
@@ -231,6 +233,14 @@ public void getFeeds(@NonNull Callback<List<Feed>> callback) {
         readPipe(BackendPipes.Names.FEEDS, uri, callback);
     }
 
+    public void getOpreationsFromFeed(@NonNull Callback<List<Operation>> callback, Resource resource) {
```
Comment (pilhuhn) [inline mobile/src/main/java/org/hawkular/client/android/backend/BackendClient.java]: getOperations

File: mobile/src/main/java/org/hawkular/client/android/backend/model/Operation.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+package org.hawkular.client.android.backend.model;
+
+import org.jboss.aerogear.android.core.RecordId;
+
+import com.google.gson.annotations.SerializedName;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Operation implements Parcelable {
```
Comment (pilhuhn) [inline mobile/src/main/java/org/hawkular/client/android/backend/model/Operation.java:26]: Some operations have parameters defined. CHeck e.g. https://github.com/hawkular/hawkular-agent/blob/master/hawkular-wildfly-agent-feature-pack/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml#L619 (and the linked XSD). So in a future PR you may want to allow to edit the parameters before running the operation.

**Comment by openwarrior** [comment]
@pilhuhn Any further change needed form this PR?

File: mobile/src/main/java/org/hawkular/client/android/backend/BackendClient.java [hunk 3]
```diff
@@ -231,6 +233,14 @@ public void getFeeds(@NonNull Callback<List<Feed>> callback) {
         readPipe(BackendPipes.Names.FEEDS, uri, callback);
     }
 
+    public void getOpreations(@NonNull Callback<List<Operation>> callback, Resource resource) {
```
Comment (pilhuhn) [inline mobile/src/main/java/org/hawkular/client/android/backend/BackendClient.java:236]: Typo Opreations

**Comment by openwarrior** [comment]
@pilhuhn Already corrected in next Work. Will be hard if I correct back in this one

**Pull request merged**